packages/editor/src/store/effects/posts.js

```diff
@@ -313,8 +314,10 @@ export const refreshPost = async ( action, store ) => {
 	const postTypeSlug = getCurrentPostType( getState() );
 	const postType = await resolveSelector( 'core', 'getPostType', postTypeSlug );
 	const newPost = await apiFetch( {
-		path: `/wp/v2/${ postType.rest_base }/${ post.id }`,
-		data: { context: 'edit' },
+		path: addQueryArgs(
+			`/wp/v2/${ postType.rest_base }/${ post.id }`,
```

Comment:
I don't mind just concatenating the query string :)

Reply:
Yeah, `addQueryArgs` might be overblown for something entirely static. I'll drop it.